include/srtp_priv.h

```diff
@@ -125,6 +125,8 @@ typedef struct srtp_stream_ctx_t_ {
     uint8_t *enc_xtn_hdr;
     size_t enc_xtn_hdr_count;
     uint32_t pending_roc;
+    bool use_cryptex;
+    struct srtp_stream_ctx_t_ *next; /* linked list of streams */
```

Comment:
next was removed, I guess this slipped in while merging ?

Reply:
There have been a lot of changes in main, are you sure the merge went well ? All builds are failing.
Also the input buffer to the protect/unprotect functions is now const, not sure you should just cast that away and move the memory around.

Reply:
i think I have been too optimistic about my rebasing 😅

Wil try to get a look next week

Reply:
Yeah, I had a quick attempt a few weeks ago and understood that this was not an easy merge :( . Let me know if you want to discuss anything.